tests/sqllogictests/suites/query/window_function/window_wisconsin.test

```diff
@@ -145,7 +145,7 @@ SELECT ntile(5) OVER (ORDER BY ten, four) nn FROM tenk1 ORDER BY ten, four, nn
 
 # cume_dist
 query I
-SELECT CAST(cume_dist() OVER (PARTITION BY four ORDER BY ten RANGE BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING)*10 as INT) FROM tenk1 WHERE unique2 < 10 order by four, ten

```

Comment:
if the frame is not unbounded, we should raise error or ?

Reply:
Will ignore frame bound in sql. 

Same behavior with lead lag.